client/components/App.jsx

```diff
@@ -1,46 +1,133 @@
 import React from 'react';
 import styles from './styles.css';
-import Grey from './Grey'
-import ProfileImage from './ProfileImage';
-import TrackImage from './TrackImage';
 
-import LikesLabel from './LikesLabel';
-import PlaysLabel from './PlaysLabel';
-import CommentsLabel from './CommentsLabel';
-import RepostsLabel from './RepostsLabel';
-
-import StatsBar from './StatsBar';
-
-import TrackListItem from './TrackListItem';
-
-import { generateSongs } from '../dataGenerators/song.mjs';
-import { generateUsers } from '../dataGenerators/user.mjs';
-import { generatePlaylists } from '../dataGenerators/playlist.mjs';
 import TrackList from './TrackList';
 import PlaylistList from './PlaylistList';
 
 import BubbleList from "./BubbleList";
-import LikesListHeader from './LikesListHeader';
+import TracksHover from "./TracksHover";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.fetchData = this.fetchData.bind(this);
+
     this.state = {
-      songs: generateSongs(3),
-      users: generateUsers(9),
-      playlists: generatePlaylists(3),
+      songs: [],
+      users: [],
+      playlists: [],
+      currentlySelected: { num_likes: undefined }
     }
-    console.log(this.state.users);
-    console.log(this.state.songs);
+
+    this.selectSong = this.selectSong.bind(this)
   }
+
+
+  selectSong(song) {
+    console.log(song);
+    this.setState({ currentlySelected: song })
+  }
+
+
+  // OLD STATIC DATA GENERATION 
+  // fetchData(callback) {
```

Comment:
if this was from the old data we shouldn't code anymore there

Reply:
Removed.